lndclient/router_client.go

```diff
@@ -64,10 +64,14 @@ type SendPaymentRequest struct {
 	// are only processed when the Invoice field is empty.
 	Invoice string
 
-	MaxFee          btcutil.Amount
-	MaxCltv         *int32
-	OutgoingChannel *uint64
-	Timeout         time.Duration
+	MaxFee  btcutil.Amount
```

Comment:
nb: Now that you're adding to this code it is perhaps a good opportunity to extend the documentation for other public fields as well :)

Reply:
commit added